stetho-repl-js/src/main/java/com/facebook/stetho/inspector/console/js/JavaScriptRuntimeBuilder.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.console.js;
```

Comment:
To avoid confusion, can we migrate these packages to `com.facebook.stetho.rhino`?  They ideally shouldn't depend on package private access into the main stetho module